R/utils_helpers.R

```diff
@@ -1,15 +1,35 @@
 #' GT Column Names
 #' 
-#' @param nm A vector of column names
+#' @param col_names A vector of column names
+#' @param col_total A vector of column totals
 #' 
 #' @description The function creates the labels for each column using the total function so the columns are now NAME N= X
 #' @export
 #' @keywords tabGen_repro
 #' 
 #' @return A character object of class \code{from_markdown}.
-# get column names with N
-col_for_list_expr <- function(nm) {
-  nm = md(glue::glue("**{row_names_n}** <br> N={total}"))
+#' 
+#' @importFrom purrr map2
+#' @importFrom gt md
+#' @importFrom glue glue
+#' @importFrom rlang set_names
+#' 
+#' @examples 
+#' data(example_dat2, package = "tidyCDISC")
+#' 
+#' labels <- col_for_list_expr(example_dat2$col_names, example_dat2$col_totals)
+#' labels
+#' 
+#' if (interactive()) {
```

Comment:
Is there a reason the `if(interactive())` is in the middle of the example? Is there any benefit to compiling only half of the example and not the other half? Just wondering if the entire example be encased in the `if(interactive())`?

Reply:
Yes, while there may not be benefit for CRAN documentation, there is benefit for other documentation like the `pkgdown` website where the code gets compiled. You can see the output of the `labels` that are getting created at least.

Reply:
Okay, as long as it's somewhat useful. I'll merge this PR and then update my cran comments before re-submitting! Thanks @Jeff-Thompson12